### PR TITLE
[REF] web: measures on report views

### PR DESCRIPTION
**Pull request by jpp-odoo**
The report views had shared behaviour on the way they compute and show
the measures. This behaviour was put in common, on a template and a
utility function.

### CODE REVIEW

File: addons/web/static/src/views/view.xml [hunk 1]
```diff
@@ -5,4 +5,32 @@
     <WithSearch t-props="withSearchProps" class="o_action o_view_controller"/>
   </t>
 
+  <t t-name="web.ReportViewMeasures" owl="1">
+    <Dropdown t-on-dropdown-item-selected="onMeasureSelected" togglerClass="'btn btn-primary'">
```
Comment (Polymorphe57) [inline addons/web/static/src/views/view.xml]: t-on-dropdown-item-selected="onMeasureSelected" does not seem to be at the right place

File: addons/web/static/src/views/view.xml [hunk 2]
```diff
@@ -5,4 +5,32 @@
...
+                    payload="{ measure: measure_value.name }"
+                    parentClosingMode="'none'"
+                    t-if="measure_value.name !== '__count'"
+                    >
+                    <t t-esc="measures[measure].string"/>
+                </DropdownItem>
+                <div t-if="measure_last &amp;&amp; Object.keys(measures).length > 1" role="separator" class="dropdown-divider"/>
+            </t>
+            <t t-set="countMeasure" t-value="measures['__count']"/>
+            <DropdownItem class="o_menu_item dropdown-item"
```
Comment (Polymorphe57) [inline addons/web/static/src/views/view.xml]: why is it not created in the foreach?

File: addons/web/static/src/views/graph/graph_model.js [hunk 3]
```diff
@@ -1,6 +1,5 @@
 /** @odoo-module **/
```
Comment (Polymorphe57) [inline addons/web/static/src/views/graph/graph_model.js:1]: Commit message:

[REF] web: measure menu for reporting views

The reporting views (cohort, graph, and pivot) did compute and show the measures in similar but slightly different ways. The present commit harmonize their behavior and factorize the code via a common template and a utility function.

**Comment by Polymorphe57** [comment]
robodoo rebase-ff r+

**Comment by jpp-odoo** [comment]
@robodoo retry

**Pull request closed without merging**